static/src/javascripts/bower.json

```diff
@@ -36,7 +36,7 @@
       "lodash-amd": "bower_components/lodash-amd/compat/**",
       "domready": "bower_components/domready/ready.js",
       "when": "bower_components/when/es6-shim/Promise.js",
-      "video.js": "bower_components/video.js/dist/video.min.js",
+      "video.js": "bower_components/video.js/dist/video.js",
```

Comment:
[There was actually another place to change it](https://github.com/guardian/frontend/commit/4c13e708a35803dac3090876eb4ef7818ae4db0c)